internals/overlord/servstate/manager_test.go

```diff
@@ -902,91 +730,74 @@ PEBBLE_ENV_TEST_PARENT=from-parent
 }
 
 func (s *S) TestActionRestart(c *C) {
```

Comment:
A little naughty drive by improvement. I still got timing related failures here, so I improved the test.